bin/gobblin-yarn.sh

```diff
@@ -50,9 +50,10 @@ function stop() {
 
 FWDIR="$(cd `dirname $0`/..; pwd)"
 FWDIR_LIB=${FWDIR}/lib
-FWDIR_CONF=${FWDIR}/conf
+FWDIR_CONF=${FWDIR}/yarn/conf
```

Comment:
Why is this yarn/conf not conf/yarn.  Especially when the yarn config files are at conf/yarn